modules/widgets/image-widget.php

```diff
@@ -70,7 +70,10 @@ public function widget( $args, $instance ) {
 
 		if ( '' != $instance['img_url'] ) {
 
-			$output = '<img src="' . esc_attr( $instance['img_url'] ) .'" ';
+			$output = '<img src="' . esc_url( jetpack_photon_url( $instance['img_url'], array(
```

Comment:
Need to add `( method_exists( 'Jetpack', 'is_module_active' ) && Jetpack::is_module_active( 'photon' ) )` conditional.